dcps/internetgateway1/internetgateway1.go

```diff
@@ -1268,7 +1268,7 @@ func (client *WANCommonInterfaceConfig1) GetMaximumActiveConnections() (NewMaxim
 	return
 }
 
-func (client *WANCommonInterfaceConfig1) GetTotalBytesSent() (NewTotalBytesSent uint32, err error) {
+func (client *WANCommonInterfaceConfig1) GetTotalBytesSent() (NewTotalBytesSent uint64, err error) {
```

Comment:
Note the comment near the top of the file: this file is generated. So this change would get lost upon next regenerating this file.

See the README.md and also how the code is generated:
https://github.com/huin/goupnp/blob/master/gotasks/specgen_task.go#L40

Right now, the "out variable" returned by the interface according to the UPnP XML spec is "ui4". You can see this by downloading http://upnp.org/specs/gw/UPnP-gw-IGD-TestFiles-20010921.zip and looking inside at `xml test files/service/WANCommonInterfaceConfig1.xml`, then looking for `GetTotalBytesSent` (at line 192 in the XML) and looking at the out variable's relatedStateVariable `TotalBytesSent` (at line 79 in the XML).

In order to get the code generator to work, you'll need to make a few changes:

1) Add your new type mapping to this map: https://github.com/huin/goupnp/blob/master/gotasks/specgen_task.go#L369
2) Add a DCPHackFn to the struct value here: https://github.com/huin/goupnp/blob/master/gotasks/specgen_task.go#L41 - you will need to have the hack function delve into the parsed XML data that it's given, and change the underlying type of the variables involved to point at your new type.